packages/reports/addon/components/common-actions/schedule.ts

```diff
@@ -134,88 +144,105 @@ export default class ScheduleActionComponent extends Component<Args> {
       return true;
     }
 
-    return !(this.localDeliveryRule?.hasDirtyAttributes && this.isRuleValid);
+    return !(this.localDeliveryRule?.hasDirtyAttributes && this.areRulesValid);
   }
 
   /**
    * @property {Boolean} attemptedSave
    */
   @tracked attemptedSave = false;
 
+  /**
+   * @action addNewRule
+   */
+  @action
+  addNewRule() {
+    this.localDeliveryRule = this._createNewDeliveryRule();
```

Comment:
What does `localDeliveryRule` do now that we can have multiple local rules? Should be this `currentDisplyedRule`?

Reply:
Yeah that would be a better name, I'll swap it.